some.js

```
@@ -0,0 +1,22 @@
+
+const myfunction= ()=> {
+console.log("Hello, world!")
+return 42;
+} 
+
+myfunction( );
+
+
+// Mocha Test with issues
+const assert = require('assert');
+describe('MyFunction', function() {
```

ESLint

[error] 'describe' is not defined.

(no-undef)